csharp/tests/Integration/GetAndSet.cs

```diff
@@ -57,13 +62,14 @@ public async Task GetReturnsEmptyString()
         await GetAndSetValues(client, key, value);
     }
 
-    [Test]
+    [Fact]
     public async Task HandleVeryLargeInput()
     {
         // TODO invesitage and fix
         if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
         {
-            Assert.Ignore("Flaky on MacOS");
+            //"Flaky on MacOS"
```

Comment:
Is there a way in xunit to mark test as skipped? on `return` it marked as passed

Reply:
It is in v3 (which is still in alpha) or could be done by NuGet https://github.com/AArnott/Xunit.SkippableFact/issues/21 (in fact example presented in XUnit repo https://github.com/xunit/samples.xunit/tree/main/DynamicSkipExample, but to use it we need to copy code in repo). 
No sure value is more than added unnecessary dependencies/extension code required

Reply:
Agree, we can keep this as is. Hope to fix those tests soon